### ISSUE TITLE
[bug] html graph unusable for qt

### ISSUE THREAD

**Issue by ericLemanissier** [conan-io/conan#15716]
### Environment details

* Operating System+version: WSL
* Compiler+version: gcc11
* Conan version: 2.1.0
* Python version: 3.10.12

### Steps to reproduce

call `conan graph info --requires=qt/6.6.1@ --format=html >graph.html`
Ignoring the fact that the answer finished with `Run conan graph info with your recipe and add --format=html` which is the command invoked in the first place.
The problem is that the resulting graph is unusable:
![image](https://github.com/conan-io/conan/assets/1926390/0ef74153-4225-45d5-8fac-9832cf01056f)
Zooming just make this line disappear. There is no error visble in chrome's js console or network analyzer

### Logs

```

======== Computing dependency graph ========
Graph root
    cli
Requirements
    autoconf/2.71#00a1e46d8ba5baaf7f10d64c1a6a0342 - Cache
    automake/1.16.5#058bda3e21c36c9aa8425daf3c1faf50 - Cache
    brotli/1.1.0#d56d7bb9ca722942aba17369cb5c0519 - Cache
    bzip2/1.0.8#457c272f7da34cb9c67456dd217d36c4 - Cache
    double-conversion/3.3.0#33321c201741cc32b51169c6d2d05e60 - Cache
    expat/2.6.0#2f92fb3cf504e9731aba6d20dcd641bf - Cache
    fontconfig/2.15.0#15a13e5b6d5931ff0144f92b579f6dd5 - Cache
    freetype/2.13.2#fa35264c88edc6421a850d08c67630f4 - Cache
    glib/2.78.1#6da549beee7208bd9fad0e6141581e8e - Cache
    harfbuzz/8.3.0#7e7ea00066eaaa818d469d24dbc3a106 - Cache
    icu/74.2#129a9b08abace8b2c748a567fc9b843a - Cache
    libelf/0.8.13#4f70a3555809ae50cc8add44f0f84288 - Cache
    libffi/3.4.4#35eb63842b505824b70aedc1baefc916 - Cache
    libiconv/1.17#73fefc1b696e069df90fd1d18aa63edd - Cache
    libmount/2.39#9b2a81545c28668d2281639cc1657dd5 - Cache
    libpng/1.6.42#fdba4ebf89df3c55d6db0145a2a66a24 - Cache
    libpq/15.4#cbae5e1ee85bd5e959e039e00307e8b1 - Cache
    libselinux/3.5#776f35e238362c6b893220f437cb89ae - Cache
    libtool/2.4.7#08316dad5c72c541ed21e039e4cf217b - Cache
    libxml2/2.12.3#071b32ddc1353a239d3697319c68beca - Cache
    m4/1.4.19#b38ced39a01e31fef5435bc634461fd2 - Cache
    odbc/2.3.11#84d32d803c970271eb2fc58d3e3cc2b5 - Cache
    openssl/3.2.1#39bd48ed31f1f3fbdcd75a0648aaedcf - Cache
    pcre2/10.42#74a354d74d291503aadffe6453d976f5 - Cache
    qt/6.6.1#dbfcf92fc905fce82a3642de76f06987 - Cache
    sqlite3/3.45.0#3a7911478ffd14e7acc694ac113449b6 - Cache
    util-linux-libuuid/2.39.2#339f80aea7e5cf507b5b9839a128909f - Cache
    wayland/1.22.0#44a1f9bf1092bea2f3d38e50db225ea6 - Cache
    xkbcommon/1.5.0#9a5d6acb805f1f7d5c3162dfd45fa23b - Cache
    xkeyboard-config/system#57dd5c08e72ab06bc2ee0e756c0a2c41 - Cache
    xorg/system#c7039fe88e71c34398de15694304f0f0 - Cache
    zlib/1.3.1#af8f2ef3d8e0fddf1975281d7fe43bf6 - Cache
Build requirements
    autoconf/2.71#00a1e46d8ba5baaf7f10d64c1a6a0342 - Cache
    automake/1.16.5#058bda3e21c36c9aa8425daf3c1faf50 - Cache
    bzip2/1.0.8#457c272f7da34cb9c67456dd217d36c4 - Cache
    flex/2.6.4#e35bc44b3fcbcd661e0af0dc5b5b1ad4 - Cache
    glib/2.78.1#6da549beee7208bd9fad0e6141581e8e - Cache
    gnu-config/cci.20210814#dc430d754f465e8c74463019672fb97b - Cache
    gperf/3.1#a9335f8833b550f10f655fd243c72978 - Cache
    libelf/0.8.13#4f70a3555809ae50cc8add44f0f84288 - Cache
    libffi/3.4.4#35eb63842b505824b70aedc1baefc916 - Cache
    libmount/2.39#9b2a81545c28668d2281639cc1657dd5 - Cache
    libselinux/3.5#776f35e238362c6b893220f437cb89ae - Cache
    m4/1.4.19#b38ced39a01e31fef5435bc634461fd2 - Cache
    meson/1.2.2#aace9dcc1db58fa42ecb5292f724092d - Cache
    meson/1.2.3#7d2bfe1bfc0c4edd9579e812344d5490 - Cache
    ninja/1.11.1#77587f8c8318662ac8e5a7867eb4be21 - Cache
    pcre2/10.42#74a354d74d291503aadffe6453d976f5 - Cache
    pkgconf/2.0.3#f996677e96e61e6552d85e83756c328b - Cache
    pkgconf/2.1.0#27f44583701117b571307cf5b5fe5605 - Cache
    zlib/1.3.1#af8f2ef3d8e0fddf1975281d7fe43bf6 - Cache
Resolved version ranges
    openssl/[>=1.1 <4]: openssl/3.2.1
    zlib/[>=1.2.11 <2]: zlib/1.3.1
Graph error
    Version conflict: Conflict between expat/2.5.0 and expat/2.6.0 in the graph.
Conflict originates from wayland/1.22.0

Run conan graph info with your recipe and add --format=html to inspect the graph errors in an easier to visualize way.
ERROR: Version conflict: Conflict between expat/2.5.0 and expat/2.6.0 in the graph.
Conflict originates from wayland/1.22.0

Run conan graph info with your recipe and add --format=html to inspect the graph errors in an easier to visualize way.
```

**Comment by ericLemanissier** [comment]
an instance of such graph can be downloaded from https://github.com/ericLemanissier/proof-of-conan/actions/runs/7990062594/artifacts/1263026833
It was created by using `conan graph info -s:a compiler.cppstd=17 --version 6.6.1 recipes/qt/6.x.x -o */*:shared=True -c tools.system.package_manager:mode=install -c tools.system.package_manager:sudo=True  --format=html >graph.html`

**Comment by AbrilRBS** [comment]
Thanks for the report, I've narrowed it down, will submit a fix for it tomorrow :)

**Comment by memsharded** [comment]
https://github.com/conan-io/conan/pull/15724 merged to fix that, it will be in next release 2.2
